app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class OrdersController < ApplicationController
+  before_action :authenticate_user!
+  before_action :authorize_order!
+  before_action :set_menu, only: %i[new create]
+  before_action :set_new_facade, only: %i[new create]
+
+  def new
+    @order = Order.new
+  end
+
+  def create
+    @order = Orders::CreateService.new(current_user, order_params).call
+    return render :new unless @order.persisted?
```

Comment:
add an empty line before and let's have NewFacade and CreateFacade where you will have `order` method